bin/run-in-docker

```diff
@@ -4,7 +4,7 @@
 # It is meant to allow development and testing without repeatedly rebuilding the docker image.  To
```

Comment:
I think this script and `bin/test-in-docker` will go away with a more thorough review, but wanted to focus on getting things running initially, with a more careful review later on.